### ISSUE TITLE
Chart version 0.1.14 is broken

### ISSUE THREAD

**Issue by arapulido** [kubernetes-incubator/service-catalog#1962]
It looks like the 0.1.14 version of the apiserver removed the --enable-admission-plugins flag, so current version of the chart is broken, as it uses this flag.

Pod logs:

```
k logs default-service-catalog-catalog-apiserver-86696d5c57-4zdn4 -c apiserver
Error: unknown flag: --enable-admission-plugins

The main API entrypoint and interface to the storage system. The API server is
also the focal point for all authorization decisions.

Usage:
  apiserver [flags]
```

**Comment by MHBauer** [comment]
v0.1.15 is pushed. We do not have any known way to fix v0.1.14 at the moment.

**Comment by jeremyrickard** [comment]
@arapulido this should be resolved by the 0.1.15 release that occured on Friday. You will need to do a helm update to get the latest chart, but it should be working after that!

**Comment by prydonius** [comment]
v0.1.15 worked for me :) - thanks for the speedy fix!
